extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -54,6 +54,7 @@ export const UIType = {
   DESKTOP_PANELS: 1, // Default desktop UI.
   DESKTOP_FULLBLEED: 2, // Desktop UI if landscape mode is enabled.
   VERTICAL: 3, // Vertical scrolling versions, for search engine bots indexing.
+  DESKTOP_ONE_PANEL: 4, // Experiment for new default desktop UI.
```

Comment:
Nits:
- a-z
- plz use a comment that describes the experience more than the state of the project. Feel free to rewrite the `DESKTOP_PANELS` comment if that helps too